app/assets/stylesheets/custom.scss

```diff
@@ -3,6 +3,22 @@
 
 $gray-medium-light: #eaeaea;
```

Comment:
please move to another file and call it back here

Reply:
done